etc/helm/pachyderm/templates/NOTES.txt

```diff
@@ -10,6 +10,10 @@ pachctl config set active-context local
 pachctl port-forward
 {{- end -}}
 
+{{- if eq .Values.pachd.logFormat "text" }}
+WARNING: setting log format to text is not supported. Defaulting to json
+{{- end -}}
```

Comment:
I think I'd warn that it's going away in 2.4 and not remove it in 2.3.  Seems odd for a patch release to make a change this large, even if "text" isn't that great.

Reply:
Ahh okay, can do!